rfc-editor/rfc8794.xml

```diff
@@ -2909,82 +2635,45 @@ be appropriate to change the reference to the newer version?
     </author>
     <date year="2011"></date>
   </front>
-<seriesInfo name="ISO/IEC" value="9899:2011" />
+  <seriesInfo name="ISO/IEC" value="9899:2011"></seriesInfo>
```

Comment:
This ISO has been reviewed in 2018.

Reply:
As I said in the email, [C18 is pretty much C11 with a few fixes](https://en.wikipedia.org/wiki/C18_(C_standard_revision)). People usually refer to C11 and not C18. So I prefer to keep 2011.

Reply:
no problem at all, it was just to have current references in the text (I personally use often an even older C myself ;-)